scripts/configure-netrc-for-mapbox-sdk-installation.sh

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
+
+# Mapbox SDK Configuration Script for iOS and Android
+# ===================================================
+#
+# Purpose:
+# --------
+# This script configures the development environment to download Mapbox SDKs 
+# for both iOS and Android builds. We use Mapbox to display maps in the App. As Mapbox SDKs 
+# are closed-sourced, we need to authenticate with Mapbox during the download.
+#
```

Comment:
> It would be good if people only have to set this up if they want to test distance requests / map related features.

This is an interesting idea, but I think it won't be easy. Unlike JavaScript code, Native code needs to be compiled for the App to function and we cannot choose dynamically which Native code to include or not.

I believe it's still possible to achieve this by editing `Podfile` or `build.grade` to introduce different configurations based on the value of an environment variable, for example.

But it might get complicated. I gonna finish the basic implementation first and then we can discuss if this is something we want to do.

but as soon as this PR gets merged, contributors need to setup the token (we can also create a new short script to check if the token is property set when engineers run `npm run android` or `npm run ios` to provide useful information if the token is not set.